Tiltfile

```
@@ -113,6 +113,28 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+addon_settings_files = []
+
+# Create a data structure to hold information about addons.
+addons = {
+    "test-extension": {
+        "context": "./test/extension",
+        "image": "gcr.io/k8s-staging-cluster-api/test-extension",
+        "container_name": "extension",
+        "binary_name": "extender",
```

```suggestion
        "binary_name": "extension",
```
Looks like it doesn't matter, but can we call this extension like in the test-extension/Dockerfile?

Did extender work? Looks like /extension is used in the YAML file (but maybe it's just overwritten later IDK)

---

This should be overwritten in tilt-prepare, but I'll change it to extension here.

---

If it's overwritten anyway and thus not used, shouldn't we drop the config parameter?

(would make the configuration simpler, and if we don't use it there shouldn't be a reason to have it)